src/Runners/PHPUnit/ResultPrinter.php

```diff
@@ -107,7 +103,6 @@ class ResultPrinter
     public function __construct(LogInterpreter $results)
     {
         $this->results = $results;
-        $this->timer = new Timer();
```

Comment:
There is no need to instantiate Timer for our usage, see

1. PHP-Timer 2: https://github.com/sebastianbergmann/php-timer/blob/2.1.2/src/Timer.php#L92
1. PHP-Timer 3: https://github.com/sebastianbergmann/php-timer/blob/3.1.4/src/Timer.php#L136
1. PHP-Timer 4: https://github.com/sebastianbergmann/php-timer/blob/4.0.0/src/ResourceUsageFormatter.php#L35
1. PHP-Timer 5: https://github.com/sebastianbergmann/php-timer/blob/5.0.0/src/ResourceUsageFormatter.php#L35